src/Tribe/API.php

```diff
@@ -92,16 +103,38 @@ public static function deleteEvent( $eventId, $force_delete = false ) {
 		/**
 		 * Used by createEvent and updateEvent - saves all the various event meta
 		 *
-		 * @param int   $event_id The event ID we are modifying meta for.
-		 * @param array $data     The meta fields we want saved.
-		 * @param       WP_Post   The event itself.
+		 * @param int     $event_id The event ID we are modifying meta for.
+		 * @param array   $data     The meta fields we want saved.
+		 * @param WP_Post $event    The event post, itself.
 		 *
+		 * @return bool
 		 */
 		public static function saveEventMeta( $event_id, $data, $event = null ) {
 			$tec = Tribe__Events__Main::instance();
 
+			$raw_data = $data;
+
 			$data = self::prepare_event_date_meta( $event_id, $data );
 
+			if ( is_wp_error( $data ) ) {
+				/**
+				 * Hook fired when saving or updating event meta fields failed due to detection of invalid data.
+				 *
+				 * Example of invalid data is an EventStartMinute of `60`, since it should be 0-59.
+				 *
+				 * @param int     $event_id The event ID we are modifying meta for.
+				 * @param array   $raw_data The meta fields we tried to send.
+				 * @param WP_Post $event    The event itself.
+				 *
+				 * @since TBD
+				 */
+				do_action( 'tribe_events_event_save_failed_invalid_meta', $event_id, $raw_data, $event );
+
+				return false;
+			} else {
+				unset( $raw_data );
```

Comment:
Don't think this `else` block is required with the content inside as the function is completed few lines afterwards and variable is no longer available after that.